src/App.tsx

```diff
@@ -17,18 +39,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                handleFilterChange={setSelectedFilter}
```

Comment:
The `handleFilterChange` function should ensure that the value being set matches the `Filters` enum values. Currently, it casts the event target value directly, which might not align with the enum. Consider mapping the string values to the enum values explicitly.